### DISCUSSION TITLE
animations on hover:

### DISCUSSION THREAD

**Discussion by dajoeberlei** [tailwindlabs/tailwindcss#2093] [Help]
Hi,

Just checked out tailwindcss1.6. The animations are awesome!
I tried to get it running on for example hover. I was trying to make a svg pulse with hover:animate-pulse but that unfortunately doesnt work. Is there a way to implement that? :-)

thanks,
Dajo

**Comment by adamwathan** [comment]
Hey! Have to enable the hover variant in your config:

```js
// tailwind.config.js
module.exports = {
  variants: {
    animation: ['responsive', 'hover']
  }
}
```
